lib/kubernetes-deploy/deploy_task.rb

```diff
@@ -41,11 +41,16 @@
 require 'kubernetes-deploy/cluster_resource_discovery'
 require 'kubernetes-deploy/template_sets'
 require 'kubernetes-deploy/deploy_task_config_validator'
+require 'kubernetes-deploy/resource_deployer'
+
+require 'krane/concerns/template_reporting'
 
 module KubernetesDeploy
   # Ship resources to a namespace
   class DeployTask
     extend KubernetesDeploy::StatsD::MeasureMethods
+    include Krane::TemplateReporting
+    delegate :logger, to: :@task_config
```

Comment:
`Krane::TemplateReporting` expectes to be able to access the logger as `logger` instead of `@logger`. I can convert this entire file to use `logger` over `@logger` if desired.